src/platforms/javascript/guides/react/index.mdx

```diff
@@ -164,12 +164,53 @@ export default Sentry.withProfiler(App);
 
 Learn more about using and customizing the React Profiler in [Profiler](components/profiler/).
 
-If you are using `react-router`, we provide `react-router` instrumentation you can use alongside Performance Monitoring with `@sentry/tracing`. Please see the [React Router Integration documentation](configuration/integrations/react-router/) for more details on setting it up.
-
 Performance data is transmitted using a new event type called “transactions”, which you can learn about in [Distributed Tracing](/product/performance/distributed-tracing/). To capture transactions, you must install the performance package and configure your SDK to set the <PlatformIdentifier name="traces-sample-rate" /> configuration to a nonzero value. The example configuration above will transmit 100% of captured transactions; lower this value in production to avoid consuming your quota too quickly.
 
 Learn more about sampling in [Filtering Events Reported to Sentry](configuration/filtering/).
 
+## React Router
+
+_(New in version 5.21.0)_
+
+React Router support is included in the `@sentry/react` package since version `5.21.0`. We support integrations for React Router 3, 4 and 5.
```

Comment:
```suggestion
Sentry's performance instrumentation includes support for React Router 3, 4 and 5.
```

Since this is directly underneath the "New in version 5.21.0," I think we can safely omit mention of that here.